airbyte-notification/src/main/java/io/airbyte/notification/CustomeriolNotificationClient.java

```diff
@@ -29,12 +30,15 @@ public class CustomeriolNotificationClient extends NotificationClient {
   // - receiver email
   // - customer.io identifier email
   // - customer.io TRANSACTION_MESSAGE_ID
-  private static final String SENDER_EMAIL = "Airbyte Notification <no-reply@airbyte.io>";
-  private static final String TRANSACTION_MESSAGE_ID = "6";
+
+  // DEFAULT_TRANSACTION_MESSAGE_ID is currently unused but the template can be used for any generic
+  // messaging.
+  // private static final String DEFAULT_TRANSACTION_MESSAGE_ID = "6";
```

Comment:
maybe we can clean up these comments a bit to mention how customer.io and the message ids are related

Reply:
lmk if there's any other info you want me to add in the comments

Reply:
thanks, I think the new docs look great!